extension.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const vscode = require('vscode');
+const utilities = require('./lib/utilities.js');
+
+
+function activate(context) {
+    console.log('Congratulations, your extension "vsc-extension-ansible" is now active!');
+    var outputChannel = vscode.window.createOutputChannel("VSCode extension for Ansible");
+
+    var playbook = vscode.window.activeTextEditor.document.fileName;
```

Comment:
i don't quite understand what's the purpose of this line here, it will be run only once, when extension is activated.

Reply:
fixed